.vscode/launch.json

```diff
@@ -8,11 +8,13 @@
             "name": "Launch VS Code Extension",
             "type": "extensionHost",
             "request": "launch",
-            "args": ["--extensionDevelopmentPath=${workspaceFolder}/keith-vscode"],
+            "args": [
+                "--extensionDevelopmentPath=${workspaceFolder}/keith-vscode",
+				"--extensionDevelopmentPath=${workspaceFolder}/../klighd-vscode/applications/klighd-vscode"
+            ],
```

Comment:
Does not belong to the default extension launch config, maybe as a separate one for launching both extensions together in a combined workspace